README.md

```diff
@@ -1,28 +1,108 @@
-# robotont_driver
+# robotont\_driver
 ROS driver package for robotont.
 
 This package handles low-level communication between ROS and robotont hardware.
 
-## Visualizing the robot's movement on Rviz:
+[![Build Status](https://travis-ci.org/robotont/robotont_driver.svg?branch=melodic-devel)](https://travis-ci.org/robotont/robotont_driver)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-### 1. Run fake_driver.launch
-* ```roslaunch robotont_driver fake_driver.launch```
-* Make sure that fixed frame is set to "odom"!
+## 1. Starting the driver
+### Real robot
+On a real robot, the driver is started automatically via a system service. You can perform several checks to verify that the driver is running properly.
 
-### 2. Moving the robot using teleop twist keyboard
-* Open a new terminal window: <br/>
-```rosrun teleop_twist_keyboard teleop_twist_keyboard.py```
-* If teleop twist keyboard is not installed:<br/>
-```sudo apt-get install ros-melodic-teleop-twist-keyboard```
+To check the status of the service:
+```bash
+systemctl status clearbot.service
+```
+
+If the driver node is running you should also see '/robotont/driver\_node' in the output of:
+```bash
+rosnode list
+```
+
+In case any of the above checks should fail try restarting the service (see the [robotont\_support](https://github.com/robotont/robotont_support) package) or launch the driver node manually:
+```bash
+roslaunch robotont_driver driver_basic.launch
+```
+
+### Fake driver
+
+In case you don't have a real robot nearby, you can run this simple dummy node that subscribes to velocity commands and publishes odometry. This driver does not know anything about physics and performs a simple integration. To start the fake driver with RViz visualization:
+```bash
+roslaunch robotont_driver fake_driver.launch
+```
+*Make sure that the fixed frame is set to "odom"!*
```

Comment:
Make sure that **in RViz** the fixed frame is set to "odom"!

Reply:
Kas selle võiks nüüd eemaldada, sest launch file hoolitseb frame eest?

Reply:
tegelt küll. ma isegi mõtlesin sama, aga siis mõtlesin, et las olla. ühesõnaga, kui rida maha võtta, siis on ka minu arust OK

Reply:
tegelikult ikka EI. see lause peab olema, sest description kimbu conf ei tea midagi odom freimist ja laaditakse RViz konf, kus FixedFrame on base_link.
Seega hetkel peaks see lause olema hoopis: Make sure to change the fixed frame in Rviz to "odom"!